velox/vector/tests/VectorTest.cpp

```diff
@@ -2587,3 +2587,26 @@ TEST_F(VectorTest, toCopyRanges) {
   rows.updateBounds();
   testRoundTrip();
 }
+
+TEST_F(VectorTest, rowCopyRanges) {
+  RowVectorPtr RowVectorDest = makeRowVector(
```

Comment:
@laithsakka @kagamiori Variable names should start lower case: RowVectorDest -> rowVectorDest